redirects.yaml

```diff
@@ -395,84 +395,116 @@ redirects:
-  - source: /image-alt/
+  - source: /image-alt
     destination: https://dequeuniversity.com/rules/axe/4.4/image-alt
-  - source: /input-image-alt/
+  - source: /input-image-alt
     destination: https://dequeuniversity.com/rules/axe/4.4/input-image-alt
-  - source: /label/
+  - source: /label
     destination: https://dequeuniversity.com/rules/axe/4.4/label
-  - source: /link-name/
+  - source: /link-name
     destination: https://dequeuniversity.com/rules/axe/4.4/link-name
-  - source: /object-alt/
+  - source: /object-alt
     destination: https://dequeuniversity.com/rules/axe/4.4/object-alt
-  - source: /color-contrast/
+  - source: /color-contrast
     destination: https://dequeuniversity.com/rules/axe/4.4/color-contrast
-  - source: /definition-list/
+  - source: /definition-list
     destination: https://dequeuniversity.com/rules/axe/4.4/definition-list
-  - source: /dlitem/
+  - source: /dlitem
     destination: https://dequeuniversity.com/rules/axe/4.4/dlitem
-  - source: /list/
+  - source: /list
     destination: https://dequeuniversity.com/rules/axe/4.4/list
-  - source: /listitem/
+  - source: /listitem
     destination: https://dequeuniversity.com/rules/axe/4.4/listitem
-  - source: /layout-table/
+  - source: /layout-table
     destination: https://dequeuniversity.com/rules/axe/3.5/layout-table
-  - source: /td-headers-attr/
+  - source: /td-headers-attr
     destination: https://dequeuniversity.com/rules/axe/4.4/td-headers-attr
-  - source: /th-has-data-cells/
+  - source: /th-has-data-cells
     destination: https://dequeuniversity.com/rules/axe/4.4/th-has-data-cells
-  - source: /meta-refresh/
+  - source: /meta-refresh
     destination: https://dequeuniversity.com/rules/axe/4.4/meta-refresh
-  - source: /meta-viewport/
+  - source: /meta-viewport
     destination: https://dequeuniversity.com/rules/axe/4.4/meta-viewport
-  - source: /audio-caption/
+  - source: /audio-caption
     destination: https://dequeuniversity.com/rules/axe/3.5/audio-caption
-  - source: /video-caption/
+  - source: /video-caption
     destination: https://dequeuniversity.com/rules/axe/4.4/video-caption
-  - source: /video-description/
+  - source: /video-description
     destination: https://dequeuniversity.com/rules/axe/3.3/video-description
-  - source: /html-has-lang/
+  - source: /html-has-lang
     destination: https://dequeuniversity.com/rules/axe/4.4/html-has-lang
-  - source: /html-lang-valid/
+  - source: /html-lang-valid
     destination: https://dequeuniversity.com/rules/axe/4.4/html-lang-valid
-  - source: /valid-lang/
+  - source: /valid-lang
     destination: https://dequeuniversity.com/rules/axe/4.4/valid-lang
+  - source: /aria-name
+    destination: https://dequeuniversity.com/rules/axe/4.4
+  - source: /duplicate-id
+    destination: https://dequeuniversity.com/rules/axe/4.4/duplicate-id-active
+  - source: /heading-levels
+    destination: https://dequeuniversity.com/rules/axe/4.4/heading-order
```

Comment:
technically these three don't have exact axe equivalents:
- `aria-name` was a catchall for a bunch of `aria-*-name` audits (e.g. [`aria-meter-name`](https://dequeuniversity.com/rules/axe/4.4/aria-meter-name)). A link to the overall rules is the best we can do, I think
- `duplicate-id` was removed by axe and replaced with `duplicate-id-active` and (and somewhat less directly, `duplicate-id-aria`). The `duplicate-id` page was kept around to keep old links alive, but we've had 37 visitors to it in the last year, so seems ok to redirect it to the more modern version of the audit
- `heading-levels` was an old manual Lighthouse audit that eventually got a real `heading-order` audit from axe. It similarly had 11 visitors in the last year, so seems ok to redirect to the modern version